sphinx_doc/setup_guide.rst

```diff
@@ -1,328 +1,246 @@
+
+Project Applications 
+--------------------
+
+Clone or refresh the django-blast project::
+
+    cd <git-home>
+    git clone https://github.com/NAL-i5K/django-blast
+    
+    # Or if the django-blast repository exists:
+    cd <git-home>
+    git fetch
+
+Yum
+---
+
+Generate metadata cache::
+
+    yum makecache
+    
 Python 2.7.8
-~~~~~~~~~~~~
+------------
 
 Install necessary packages::
 
-    yum groupinstall "Development tools"
-    yum install zlib-devel bzip2-devel openssl-devel ncurses-devel sqlite-devel 
-    yum install readline-devel tk-devel gdbm-devel db4-devel libpcap-devel xz-devel python-devel
+    sudo yum -y groupinstall "Development tools"
+    sudo yum -y install zlib-devel bzip2-devel openssl-devel ncurses-devel sqlite-devel 
+    sudo yum -y install readline-devel tk-devel gdbm-devel db4-devel libpcap-devel xz-devel python-devel
 
 Install python 2.7.8 from source::
 
     wget http://www.python.org/ftp/python/2.7.8/Python-2.7.8.tar.xz  
-    xz -d Python-2.7.8.tar.xz  
-    tar -xvf Python-2.7.8.tar  
-
-    # Enter the directory:
+    tar -xf Python-2.7.8.tar  
+    
+    # Configure as a shared library:
     cd Python-2.7.8
+    ./configure --prefix=/usr/local --enable-unicode=ucs4 --enable-shared LDFLAGS="-Wl,-rpath /usr/local/lib"
 
-    # Run the configure:
-    ./configure --prefix=/usr/local
-
-    # compile and install it:
+    # Compile and install:
     make  
-    make altinstall
-
-    # Checking Python version:
-    [root@i5k ~]# python2.7 -V
-    Python 2.7.8  
-
+    sudo make altinstall
+    
+    # Update PATH:
     export PATH="/usr/local/bin:$PATH"
-
-Intall pip and virtualenv::
+    
+    # Checking Python version (output should be: Python 2.7.8):
+    python2.7 -V
+    
+Install pip and virtualenv::
 
     wget https://bootstrap.pypa.io/ez_setup.py
-    python2.7 ez_setup.py
+    sudo /usr/local/bin/python2.7 ez_setup.py
+    
     wget https://bootstrap.pypa.io/get-pip.py
-    python2.7 get-pip.py
-    pip2.7 install virtualenv
-
-Build a separated virtualenv::
+    sudo /usr/local/bin/python2.7 get-pip.py
+    
+    sudo /usr/local/bin/pip2.7 install virtualenv
 
-    # create a virtual environment called py2.7. 
-    # Activate it by source py2.7/bin/activat
+Build a separate virtualenv::
 
-    # Make dir '/path/to/i5k/virtualenv' and switch in
-    mkdir /path/to/i5k/virtualenv 
-    cm /path/to/i5k/virtualenv
-    /usr/bin/virtualenv py2.7 
+    # Make root dir for virtualenv and cd into it:
+    mkdir <virt-env>
+    cd <virt-env> 
+    
+    # Create a virtual environment called py2.7 and activate:
+    virtualenv py2.7 
     source py2.7/bin/activate
-
-RabbitMQ 
-~~~~~~~~
+    
+    # Install the project in the virtual environment:
+    cd <app-home> 
+    cp -pr <app-home> <virt-env> 
+    
+Python Modules and Packages
+---------------------------
+
+Install additional Python packages::
+
+    cd <virt-env>
+     
+    # Cut, paste and run the following bash script.
```

Comment:
Two comments here:
Would it be better to just add the script itself then have the instructions call out to source the script?
Isn't this missing the shebang? It isn't required, but is helpful.